packages/react/src/ActionList/ActionList.module.css

```diff
@@ -0,0 +1,51 @@
+/* wrapper span
+ default block */
+.ItemDescriptionWrap {
```

Comment:
I'm trying my best to chunk out the CSS for ActionList but some of my PRs will probably include unrelated CSS for my own merge conflict sanity 😄 🙌